src/simperium/channel.js

```diff
@@ -645,7 +645,7 @@ LocalQueue.prototype.resendSentChanges = function() {
 function collectionRevisions( channel, id, callback ) {
 	var expectedVersions = -1;
 	var onGhostRetrieved = function( ghost ) {
-		var version = Math.min( ghost.version, 30 );
+		var version = Math.min( ghost.version, 160 );
```

Comment:
Optional: should we add a constant for this or a comment explaining how did we come up with this number?

Reply:
well right now there's a comment in the "blame" for the source control. I'd be fine creating a constant but wanted to make as small of a change as I could. I'm not certain I understand why 30 was there even.